README.md

```diff
@@ -51,36 +55,52 @@ curl  --tlsv1.2 -sSf https://sh.rustup.rs | sh
 Other rust installation methods are available on the
 [rust website](https://forge.rust-lang.org/infra/other-installation-methods.html).
 
-To install the FFTW library on MacOS, one could use the Homebrew package manager. To install
-Homebrew, you can do the following:
+### macOS
+
+To have the required C compiler and linker you'll either need to install the full __XCode__ IDE
+(that you can install from the AppleStore) or install the __Xcode Command Line Tools__ by typing the
+following command:
 
 ```bash
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+xcode-select --install
 ```
 
-And then use it to install FFTW:
+#### Apple Silicon
 
+Concrete is currently only available for x86_64 architecture.
+To use it on Apple Silicon chip, you need to compile `concrete` for the `x86_64` architecture
+and let Rosetta2 handle the conversion.
+
+First install the needed rustc target:
 ```bash
-brew install fftw
+# Install the macOS x86_64 (you only need to do this once)
+rustup target install x86_64-apple-darwin
 ```
 
-**Note for Apple Silicon users**: Concrete is currently only available for x86 architecture.
-To use it on Apple Silicon chip, you could use an x86_64 version (more detailed information
-could be found [here](https://github.com/zama-ai/concrete/issues/65#issuecomment-902005481)).
-
-To install FFTW on a debian-based distribution, you can use the following command:
-
+Then in all the `cargo` commands that you do, add the `--target x86_64-apple-darwin` argument.
+eg:
```

Comment:
Here I would either write:
"... argument, e.g.: " or "... argument. For example:"